pages/en/blog/vulnerability/april-2024-security-releases.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2024-03-27T03:00:00.000Z
+category: vulnerability
+title:  Wednesday, April 3, 2024 Security Releases
```

Comment:
It probably doesn't matter but there's a double space here.

Reply:
```suggestion
title: Wednesday, April 3, 2024 Security Releases
```

Reply:
It was automatically generated. We'll fix it.